audiomentations/augmentations/transforms.py

```diff
@@ -110,3 +193,17 @@ def apply(self, samples, sample_rate):
         max_amplitude = np.amax(np.abs(samples))
         normalized_samples = samples / max_amplitude
         return normalized_samples
+
+class Trim(BasicTransform):
+    """
+    Apply librosa.effects.trim
```

Comment:
I would improve this docstring, so that it explains what librosa.effects.trim means/does:
"Trim leading and trailing silence from an audio signal."

Reply:
Done